charts/pulsar/templates/presto/presto-coordinator-statefulset.yaml

```diff
@@ -129,6 +129,11 @@ spec:
           {{- end }}
           command: ["sh", "-c"]
           args:
+            {{ if .Values.presto.coordinator.commandOverride }}
+            {{ with .Values.presto.coordinator.commandOverride }}
+            -{{- toYaml . | indent 12 }}
+            {{- end }}
+            {{ else }}
```

Comment:
Do we need to use `{{- else }}` and `{{- end }}` here to chomp whitespaces?

Reply:
done